theano/tensor/opt.py

```diff
@@ -1781,7 +1784,7 @@ def local_alloc_unary(node):
             # Is it really necessary to copy over stack trace here?
             # after all, T.alloc and T.cast should preserve the stack trace from x,
             # but perhaps the trace is lost in "v = node.op(x)"?
```

Comment:
I'll change the comment, but leave the actual code alone (I think copying it over like this is fine).